### PR TITLE
docs: update to show how to run the test suite with the "certsuite" cmd

### PR DESCRIPTION
**Pull request by jmontesi**

### CODE REVIEW

File: docs/test-run.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+The Certsuite binary can be built as follows:
+
+```shell
+make build-certsuite-tool
+```
+
+## Test labels
+
+The test cases cases have several labels to allow for different types of groupings when selecting which to run. These are the following:
```
Comment (sebrandon1) [inline docs/test-run.md:35]: Should we also include `all` here as an option?

File: docs/test-run.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+    See the [OCT tool](https://github.com/test-network-function/oct) for more information on how to create this DB.
+
+## Using the container image
+
+The only prerequisite for running the Test Suite in container mode is having Docker or Podman installed.
+
+### Pull the test image
+
+The test image is available at this repository [repository](https://quay.io/repository/testnetworkfunction/cnf-certification-test) and can be pulled using:
```
Comment (sebrandon1) [inline docs/test-run.md:81]:
```suggestion
The test image is available at this [repository](https://quay.io/repository/testnetworkfunction/cnf-certification-test) and can be pulled using:
```

File: docs/test-run.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+```shell
+docker pull quay.io/testnetworkfunction/cnf-certification-test
```
Comment (sebrandon1) [inline docs/test-run.md:84]: Should we specify `:latest` here?  Or add some text about pulling a desired release version?

File: docs/test-run.md (hunk 1 above)
Comment (jmontesi) [reply docs/test-run.md:35]: It's explained below:
`To select all the test cases the "all" label filter can be used.`

I think `all` is more of a label filter (that gets expanded in the code into the different categories) than a proper label.

File: docs/test-run.md (hunk 2 above)
Comment (jmontesi) [reply docs/test-run.md:81]: Done.

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**